pg_shard.c

```diff
@@ -1045,6 +1653,23 @@ CreateTemporaryTableLikeStmt(Oid sourceRelationId)
 
 
 /*
+ * CreateUniqueTableName creates a unique table name on every invocation. It takes
+ * no parameters and returns a StringInfo.
+ */
+static StringInfo
+CreateUniqueTableName(void)
```

Comment:
Turns out this was inline in another function already for the multi-shard select written by @sumedhpathak so we'll leave it as-is.